tests/Keboola/DbExtractor/AbstractSnowflakeTest.php

```diff
@@ -41,18 +45,25 @@ public function setUp()
             sprintf("USE SCHEMA %s", $this->connection->quoteIdentifier($config['parameters']['db']['schema']))
         );
 
-        $this->storageApiClient = new Client([
-            'token' => getenv('STORAGE_API_TOKEN')
-        ]);
+        $this->storageApiClient = new Client(
+            [
+            'token' => getenv('STORAGE_API_TOKEN'),
+            ]
+        );
 
         $this->setupTables();
 
         $fileSystem = new Filesystem();
-        $fileSystem->remove(__DIR__ . '/data/out');
-        $fileSystem->remove(__DIR__ . '/data/runAction/config.yml');
-        $fileSystem->remove(__DIR__ . '/data/runAction/out');
-        $fileSystem->remove(__DIR__ . '/data/connectionAction/config.yml');
-        $fileSystem->remove(__DIR__ . '/tests/data/connectionAction/out');
+        $fileSystem->remove($this->dataDir . '/out');
```

Comment:
This should be part of https://github.com/keboola/db-extractor-snowflake/pull/67